lib/xlsxir/parse_worksheet.ex

```diff
@@ -87,10 +87,14 @@ defmodule Xlsxir.ParseWorksheet do
     %{state | value_type: :value}
   end
 
+  def sax_event_handler({:startElement, _, 'is', _, _}, state, _, _),
+    do: %{state | value_type: :inline_string}
```

Comment:
I believe we can use the type value here, or change the previous method implementation to use guards instead a constant.

```suggestion
    do: %{state | value_type: :value}
```